core/src/main/java/org/apache/accumulo/core/cli/ClientOpts.java

```diff
@@ -375,6 +376,7 @@ public ClientConfiguration getClientConfiguration() throws IllegalArgumentExcept
         }
 
         @Override
+        @SuppressModernizer
```

Comment:
This is pretty nice, but I wish the suppression was more specific... like spotbugs, where you can suppress specific codes (and provide a justification inline).

Reply:
That would be nice but the annotation doesn't take any values.  If you have an idea how it could be done, it might not be bad to do a PR against https://github.com/gaul/modernizer-maven-plugin